pretext/SimplePythonData/OperatorsandOperands.ptx

```diff
@@ -14,79 +14,87 @@ hour * 60 + minute
 minute / 60
 5 ** 2
 (5 + 9) * (15 - 7)</pre>
-  <p>The tokens <c>+</c>,    <c>-</c>, and <c>*</c>, and the use of parenthesis for grouping,
-            mean in Python what they mean in mathematics. The asterisk (<c>*</c>) is the
-            token for multiplication, and <c>**</c> is the token for exponentiation.
-            Addition, subtraction, multiplication, and exponentiation all do what you
-            expect.</p>
+
+<p> Python uses special <c>characters</c> called <c>operators</c> that perform the following operations: </p>
+<ul> 
+<li> <c> + </c> addition </li>
+<li> <c> - </c> subtraction </li>
+<li> <c> * </c> multiplication </li>
+<li> <c> / </c> division </li>
+<li> <c> ** </c> exponentiation </li>
+<li> <c> () </c> grouping </li>
+<li> <c> // </c> integer division (round down division) </li>
+<li> <c> % </c> modulus (remainder) </li>
+</ul>
+
+<p> There are several other operators that we will learn in future chapters. But these are the ones that you're going to need more or less all of your assignments! </p>
+
   <program xml:id="ch2_sec7_p1" interactive="activecode" language="python">
     <input>
-print(2 + 3)
-print(2 - 3)
-print(2 * 3)
-print(2 ** 3)
-print(3 ** 2)
+print(2 + 3)   # Addition operator, expected result is 5
+print(2 - 3)   # Subtraction operator, expected result is -1
+print(2 * 3)   # Multiplication operator, expected result is 6
+print(5 / 3)   # Division operator, expected result is 1.6666666666666666
+print(3 ** 2)  # Exponentiation operator, expected result is 9
+print(3*(2+4)) # Grouping operator, changes the precedence expected result is 18 
     </input>
   </program>
-  <p xml:id="simple-python-data_index-1">When a variable name appears in the place of an operand, it is replaced with
-            the value that it refers to before the operation is performed.
-            For example, what if we wanted to convert 645 minutes into hours.
-            In Python 3, division is denoted by the operator token <c>/</c> which always evaluates to a floating point
-            result.</p>
+
+<p> The first six operators have functionality that you're most probably pretty familiar with! On the other hand, we have two more operators that are used frequently in programming but not that much in math classes.
+These are integer division, and modulus. </p>
+
+<p> In the previous example <c> 5 / 3 </c> resulted <c> 1.66666666 </c>. This is because <c>/</c> always evaluates to a floating point result. However, in certain situations, we need an integer result from division.
+<term>Integer division</term>, or the <c> // </c> operator, is division such that its results are always rounded down to the nearest integer. So for the previous example, <c> 5 // 3 </c> will result <c> 1 </c> </p>
+
+<p>The <term>modulus operator</term>, <c> % </c> works on integers (and integer expressions) and yields
```

Comment:
Do we need to specify integer expressions? I feel like its kinda confusing if we dont provide an example.

Reply:
This turns out to be incorrect. Modulus should be modulo. And more importantly, modulo operator works with floats too.